compiler/rustc_mir/src/interpret/operand.rs

```diff
@@ -160,10 +161,11 @@ pub enum Operand<Tag = ()> {
 pub struct OpTy<'tcx, Tag = ()> {
     op: Operand<Tag>, // Keep this private; it helps enforce invariants.
     pub layout: TyAndLayout<'tcx>,
+    pub span: Span,
```

Comment:
Please add a comment explaining where this comes from... is this only ever set for constants? In that case, does it point to the def.n or the sue of the const?